.github/issue_template.md

```diff
@@ -0,0 +1,61 @@
+This template includes boilerplate for different kinds of issues. Select the
+one that is most suitable, delete first level header and the other templates.
+
+# Template 1: Bug
+
+Description of the problem
+
+## Affected platforms
+
+## Repro steps
+
+1. One
+2. Two
+3. Three
+
+## Expected behavior
+
+## Actual behavior
+
+# Template 2: Suggest new API
+
+Read http://aka.ms/apireview and include description of the problem.
+
+## Rational and Usage
+
+Include some sample code that illustrates how the API will be used. If
+applicable, include the way it could be done without the API to illustrate
+how the new API makes things easier.
+
+## Proposed API
+
+Include an outline of the API. It doens't have to be complete, but enough to
+get a handle on what would be required in terms of surface area.
+
+```C#
+namespace System.Foo {
+    public class Fizz {
+        public Fizz();
+        public void int Epic(string message);
+    }
+}
+```
```

Comment:
For API refs we deliberately use the other style as this makes it easier to review large chunks of APIs. But for the issue description I don't think we care enough one way or the other :-)